scss/detail.scss

```diff
@@ -3,36 +3,47 @@
 .card {
   margin: 25px;
   padding: 25px;
+  display: flex;
 
-  .header {
-    margin-bottom: 50px;
+  .card-body{
+    order: 2;
```

Comment:
you could just define the img before the card-body on the html

Reply:
same answer as above